src/AElf.Kernel.CodeCheck/Application/ICodeCheckProposalService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace AElf.Kernel.CodeCheck.Application;
+
+public interface ICodeCheckProposalService
+{
+    void AddToReleasedProposal(Hash proposalId, Hash proposalInputHash, long height);
+    Task<List<CodeCheckProposal>> GetToReleasedProposalListAsync(Address from, Hash blockHash, long blockHeight);
```

Comment:
Maybe change to `GetReleasableProposalListAsync`.

Reply:
Got it.